index.html

```diff
@@ -425,6 +425,12 @@ <h3>
     Contributing factors represent the underlying hardware metrics contributing to the [=current pressure state=]
     and can be [=implementation-defined=].
   </p>
+  <p>
+    The <dfn>computing pressure state</dfn> steps, given [=implementation-defined=] |data| from contributing factors
```

Comment:
I think we don't need to define this as steps. We can state in prose what the implementations must do. An "adjusted pressure state" is an output of this. Proposal:

`The <dfn>adjusted pressure state</dfn> is a [=pressure state=] determined by an [=implementation-defined=] algorithm that takes as input [=source type=] and (insert other relevant inputs here). This algorithm MUST not be deterministic to ensure [=break calibration=] mitigation effectiveness.`

Reply:
If yo refer to https://www.w3.org/TR/compute-pressure/#data-delivery step 2, 
I believe you can say there:

```
Let |state:PressureState| be an [=adjusted pressure state=] given |data| and |source|.
```

I think this could be named as "steps" too but it reads better to me as prose.

https://infra.spec.whatwg.org/#algorithm-declaration

While looking at Infra, I found this part interesting (although not directly related to this PR):
https://infra.spec.whatwg.org/#algorithm-limits

Reply:
I think it is good to name the output of this step.
"adjusted pressure state" sounds like a correct term.